projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/AppointmentOutcomeService.kt

```diff
@@ -21,8 +23,17 @@ class AppointmentOutcomeService(
             contactTypeOutcomeRepository.getByTypeIdAndOutcomeCode(appointment.type.id, outcome.code)
 
         appointment.apply {
+            val dateTime = LocalDateTime.now()
             attended = outcome.attended
-            notes = outcome.notes
+            complied = if (contactTypeOutcome.outcome.outcomeCompliantAcceptable!!) "Y" else "N"
+            notes = outcome.notes?.let {
+                """
+                Comment added by ${outcome.recordedBy} on ${
+                    dateTime.format(DeliusDateTimeFormatter).substring(0, 10)
+                } at ${dateTime.format(DeliusDateTimeFormatter).substring(11, 16)}
```

Comment:
Isn't this line added by the Delius DB trigger?  Do you need to add it manually?

Reply:
When I ran the api in dev, only the note was added.  The 'created by' part was empty, see second contact for D001024 in dev.

Reply:
I assume 'notes += outcome.notes' will work?

Reply:
I believe so, you might need to stick a new line in there as well but I'm not sure.

Reply:
going to deploy my branch to dev with an update to see if the notes are created successfully